src/components/DefinitionEntry.js

```diff
@@ -67,6 +67,15 @@ export default class DefinitionEntry extends React.Component {
     return get(this.props.definition, field)
   }
 
+  ifDifferent(field, then_, else_) {
+    return this.props.otherDefinition && !isEqual(get(this.props.otherDefinition, field), this.getOriginalValue(field))
```

Comment:
hmm, this pattern obscures things a bit. Perhaps just `isDifferent` and then the caller can write their own conditional. For example, in `renderWithToolTipIfDifferent` below all the rendering etc can be eliminated in the `else` case. so it would be something like 

```javascript
  if (!this.isDifferent(field)) return content
  return ( < render lots of stuff >)
```
Seems clearer and more efficient.